src/lib/event/DY.Lib.Event.Typed.fst

```diff
@@ -52,7 +52,7 @@ let split_event_pred_params: split_function_parameters = {
     Some (tag, (tr, prin, content))
   ));
 
-  local_fun_t = trace -> principal -> bytes -> prop;
+  local_fun_t = mk_dependent_type (trace -> principal -> bytes -> prop);
   global_fun_t = trace -> principal -> string -> bytes -> prop;
```

Comment:
It is just before the `bytes` argument that is associated with this `bytes`. In fact it could make sense to bundle them and write the predicate as `trace -> principal -> (string & bytes) -> prop`, so I think the order makes sense as it is.